### PR TITLE
testing

### PR DESCRIPTION
**Pull request by sanjaysheel**
it is causes some prob lem let se

### CODE REVIEW

**Comment by bantinarvariya** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Bootstrap Example</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
</head>
<body> 
      <style>
        * {
          box-sizing: border-box;
        }
        
        body {
          font-family: Arial, Helvetica, sans-serif;
        }
        
        /* Style the header */
        header {
          background-color: #666;
          padding: 30px;
          text-align: center;
          font-size: 35px;
          color: white;
        }
        
        /* Create two columns/boxes that floats next to each other */
        nav {
          float: left;
          width: 20%;
         /* only for demonstration, should be removed */
          background: #ccc;
          padding: 20px;
        }
        
        /* Style the list inside the menu */
        nav ul {
          list-style-type: none;
          padding: 0;
        }
        
        article {
          float: left;
          padding: 20px;
          width: 70%;
          background-color: #f1f1f1;
          height: 300px; /* only for demonstration, should be removed */
        }
        
        /* Clear floats after the columns */
        section:after {
          content: "";
          display: table;
          clear: both;
        }
        
        /* Style the footer */
        footer {
          background-color: #777;
          padding: 10px;
          text-align: center;
          color: white;
        }
        
        /* Responsive layout - makes the two columns/boxes stack on top of each other instead of next to each other, on small screens */
        @media (max-width: 600px) {
          nav, article {
            width: 100%;
            height: auto;
          }
        }
        </style>
        </head>
        <body>
        <header>
          <h2>c language</h2>
        </header>
        
        <section>
          <nav>
            <ul>
              <li><a href="#">hello world</a></li>
              <li><a href="#">variable</a></li>
              <li><a href="#">Keywords & Identifier</a></li>
              <li><a href="#">C Input/Output</a></li>
              <li><a href="#">C Operators</a></li>
              <li><a href="#">C if...else</a></li>
              <li><a href="#">C for Loop</a></li>
              <li><a href="#">C do...while Loop</a></li>
              <li><a href="#">C break and continue</a></li>
              <li><a href="#">C switch...case</a></li>
              <li><a href="#">C Programming goto</a></li>
              <li><a href="#">Control Flow Examples</a></li>
              <li><a href="#">C Programming Functions</a></li>
              <li><a href="#">C User-defined Functions</a></li>
              <li><a href="#">C Function Types</a></li>
              <li><a href="#">C Recursion</a></li>
              <li><a href="#">C Storage Class</a></li>
              <li><a href="#">C Function Examples</a></li>
              <li><a href="#">C Programming Arrays</a></li>
              <li><a href="#">C Multi-dimensional Arrays</li>
              <li><a href="#">C Arrays & Function</a></li>
              <li><a href="#">C Programming Pointers</a></li>

            </ul>
          </nav>
    
          
          <article>
            <h1>hello world</h1>
             <p><code>
                #include <stdio.h> <br>
                    int main() <br>
                    { <br>
                       // printf() displays the string inside quotation <br>
                       printf("Hello, World!"); <br>
                       return 0; <br>
                    } <br>
             </code></p>
          </article>
        </section>
        
        </body>
        </html>
        
</body>

**Comment by sanjaysheel** [comment]
[c-language.txt](https://github.com/Anonvia/GetKnowledge/files/3299812/c-language.txt)


make header part c-language static and body part also make static if we make more content so it will scroll not down the page and also anchor tag also make this part static it should be scroll

thanx
